test/integration/eu-west-1/k8s-qa/cluster/terragrunt.hcl

```diff
@@ -105,7 +105,7 @@ inputs = {
       # This comment configures the renovate bot to automatically update this variable:
       # amiFilter=[{"Name":"owner-id","Values":["602401143452"]},{"Name":"name","Values":["amazon-eks-node-1.25-*"]}]
       # currentImageName=amazon-eks-node-1.25-v20230501
-      ami_id             = "ami-00313eae2f5c20b18"
+      ami_id             = "ami-04ffe0600deb5c715"
```

Comment:
Is this a 1.26 ami? If so, you forgot to update the general node group above and you need to update the filter `amiFilter` (the `"amazon-eks-node-1.25-*"`) above so that renovate can keep updating the AMIs for us.

Reply:
The ami filter needs to be `"amazon-eks-node-1.26-*"`

Reply:
Next time you can just update the ami filter and let renovate update the amis.